big_tests/tests/muc_SUITE.erl

```diff
@@ -2711,6 +2744,31 @@ user_unregisters_nick_twice(Config) ->
         ?assert_equal(<<>>, get_nick(Alice))
     end).
 
+user_cancels_registration(Config) ->
+    escalus:fresh_story(
+      Config, [{alice, 1}],
+      fun(Alice) ->
+              Form = form_helper:form(#{type => <<"cancel">>}),
+              SetIQ = escalus_stanza:iq_set(?NS_INBAND_REGISTER, [Form]),
+              Req = escalus_stanza:to(SetIQ, muc_helper:muc_host()),
+              escalus:send_iq_and_wait_for_result(Alice, Req)
```

Comment:
Actually there is an assertion inside: it has to be a result that corresponds to the request (has the same `id` attribute). this particular request is weird, as it is a no-op, but I didn't want to remove its handling from the code, because it is technically valid. It just does nothing, and the result contains nothing.

Reply:
Ok 👍 Thanks for clarifying